capstone/src/main/java/com/google/sps/CloudNLPAssociation.java

```diff
@@ -62,7 +64,7 @@ public CloudNLPAssociation(LanguageServiceClient language) {
     List<Entity> entities = response.getEntitiesList();
     ArrayList<EntitySentiment> res = new ArrayList<EntitySentiment>();
```

Comment:
Res is kind of an ambiguous name for a variable. I know that you don't want to make it too long, but maybe something like entityResponses or responses?

Reply:
Changed it to result